TextToSpeech/TextToSpeechJsonRpc.cpp

```diff
@@ -166,17 +169,56 @@ namespace Plugin {
     {
         if(_tts) {
         Exchange::ITextToSpeech::Configuration config;
-        config.ttsEndPoint = GET_STR(parameters, "ttsendpoint", "");
-        config.ttsEndPointSecured = GET_STR(parameters, "ttsendpointsecured", "");
+        std::string proxyTtsEndPoint = GET_STR(parameters, "ttsendpoint", "");
+        std::regex urlPattern(R"((https?|ftp):\/\/\S+\/\S*)",std::regex::icase);
```

Comment:
* please leave a comma after ','
* Please adhere to this style

> if (condition) {
> } else {
> }

* After a condition / loop block, please leave a new line
* TTS endpoint can have "http://" as well as "https://"
* Secure TTS endpoint can only have "https://"